routes/api/v1/favorites.js

```diff
@@ -27,4 +27,26 @@ router.post('/', (request, response) => {
     })
 });
 
+async function allFavorites() {
+  try {
+    let response = await database('favorites').select("id", "title", "artistName", "genre", "rating")
+    return response;
+  } catch(err) {
+    return err;
+  }
+}
```

Comment:
Great refactoring here.  I believe I will be able to reuse this for the feature I built `GET api/v1/favorites/:id`